public/blog/testrpc-is-now-ganache.md

```diff
@@ -0,0 +1,35 @@
+# TestRPC is now Ganache
+
+Have you been looking for the TestRPC recently and haven't been able to find it?
+
+Don't worry, it's still here! It's just now known as **Ganache**.
+
+It comes in two flavors: A fully-interactive graphical interface, and the more familiar command-line version.
+
+* [Get the graphical version here](/ganache).
+
+* For the command-line version (known as **Ganache CLI**):
+
+  ```shell
+  npm install -g ganache-cli
```

Comment:
Aside: I wonder if this whole section should be under the "Multiple ways to enjoy" heading, as that section introducing the same thing but with the addition of the library.

Reply:
Originally I had it there. @benjamincburns suggested putting the install command at the top of the post (as soon as possible basically) because it was one of the main questions this post was trying to answer (as in, "*how do I get the command line version of what used to be called the TestRPC?*") So it's up there as a kind of a TL;DR now. (If we had a fold, it would be above it.)